corehq/apps/reports/tasks.py

```diff
@@ -175,6 +175,14 @@ def saved_exports():
         export_for_group_async.delay(group_config, 'couch')
 
 
+@task(queue='background_queue', ignore_result=True)
+def rebuild_export_task(groupexport_id, index, output_dir='couch', last_access_cutoff=None, filter=None):
+    from couchexport.groupexports import rebuild_export
+    group_config = HQGroupExportConfiguration.get(groupexport_id)
```

Comment:
moved from https://github.com/dimagi/couchexport/pull/143 and changed the doc type so that proper filtering gets applied